include/mbedtls/ssl.h

```diff
@@ -313,6 +313,7 @@
 #define MBEDTLS_SSL_MSG_ALERT                  21
 #define MBEDTLS_SSL_MSG_HANDSHAKE              22
 #define MBEDTLS_SSL_MSG_APPLICATION_DATA       23
+#define MBEDTLS_SSL_MSG_CID                    25
```

Comment:
Note: though this is the value given in the draft, it hasn't been formally confirmed by IANA yet. We'll probably want to double-check that nothing changed when the draft turns into an RFC. (Nothing to change in this PR, though.)